Ports Pun Pun to Torch

comma opened the pull request:
Little guy joined EC some time ago, and now prowls maintenance of the ship.
Also fixes missing pixels of monkey hands and makes monkey butts a bit smoother.
He has bunch of random spawn points to pick from, strewn in maintenance.

![](http://i.imgur.com/AaGiHym.png)

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

Raptor1628 commented:
Please no. Stick him on centcomm if you gotta. Or hell, give him a red hardsuit and give him to the mercs.

Zero-Bits commented:
Please yes.

BlueNexus commented:
Please no.

ghost commented:
Pun pun looking sharp.

Finalsong commented:
Pun Pun can't be on the Torch. He's much too busy waging a war against Vox colonists with his cube monkey army in the ruins of the Exodus

mustafakalash commented:
Why are you okay with Runtime and Ian, but not Pun?

Runa-Dacino commented:
Pun pun for Torch!

Raptor1628 commented:
I handed runtime off to the counselor as a psych animal, not entirely unreasonable, although honestly Ian might fit better in that role, and Ian, while he may be replaced in the near future, is a small dog with a clear owner. Monkeys/chimpanzees are much harder to train, clean up after, and for that matter wouldn't be dressed in a mini-uniform and stashed in the bar. If you want a meme pet, put them on the admin Z.

Albens commented:
Can we just add him and not map him in, then?

comma commented:
He's  not in the bar in any official capacity. He's left in (random) out of place maintenance, idea is that it was a prank by some bored EC guy, as description of uniform implies.
I don't really see the point to adding him where players can't stumble onto him.

Albens commented:
Adding him but not mapping him in would just leave him for admin gimmicks or use later.

comma commented:
I'd rather have him be a thing to stumble upon once in a while in maintenance rather than not present anywhere sans for unspecified "admin gimmicks". Admin-only content is stupid.

Raptor1628 commented:
I'd prefer this go up for a vote, but if more people want the filthy monkey in, I'd prefer if he only spawned maybe one out of every three rounds.

atlantiscze commented:
I see literally no reason to bring this thing to the torch. I found it silly on the exodus, and it'll probably be even more silly on the torch. I was living under the impression that it's government operated expedition ship, not a flying zoo.

comma commented:
Eh sure if that's this big of a deal.

PsiOmegaDelta commented:
As these spawn points have weights it's just a matter of placing one on the Admin level with a custom weight to ensure Pun-Pun ends up there about 2 out of 3 times.

comma commented:
Demoacracy is neat when it picks what I want!

comma committed:
Adds Pun Pun to Torch

Little guy joined EC some time ago, and now prowls maintenance of the ship.
Also fixes missing pixels of monkey hands and makes monkey butts a bit smoother.
He has bunch of random spawn points to pick from, strewn in maintenance.

Iamgoofball commented:
Stop lying about it being used for admin gimmicks, don't be a coward, just say you don't want him in if you don't want him in

:+1:

PsiOmegaDelta commented:
Map conflicts.

comma committed:
Mapping him in
